pkg/v1/config/clientconfig.go

```diff
@@ -37,6 +37,10 @@ const (
 	// when creating a cluster.
 	FeatureFlagManagementClusterCustomNameservers = "features.management-cluster.custom-nameservers"
 	FeatureFlagClusterCustomNameservers           = "features.cluster.custom-nameservers"
+	// Custom Nameserver feature flags determine whether it is permitted to
```

Comment:
Copy paste error? I don't think you want this comment specifically as it is specific to custom nameservers.

Reply:
Good catch! It's fixed now.